module-new/src/components/inputWrapper/inputWrapper.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import { expect } from '@storybook/jest';
+import { Meta, StoryObj } from '@storybook/react';
+import { userEvent, within } from '@storybook/testing-library';
+import * as React from 'react';
+
+import { InputWrapper } from './inputWrapper.component';
+
+/** metadata */
+
+export default {
+  title: 'Components/InputWrapper',
+  component: InputWrapper,
+} as Meta<typeof InputWrapper>;
+
+/** component template */
+
+const Template: StoryObj<typeof InputWrapper> = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any -- The type discriminator on InputWrapper prevents storybook from spreading pure props on here without a cast
+  render: (props: any) => (
+    <InputWrapper {...props}>
+      <input type="text" />
+    </InputWrapper>
+  ),
+};
+
+/** stories */
+
+export const Default: StoryObj<typeof InputWrapper> = {
+  ...Template,
+  args: {
+    label: 'First Name',
+  },
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement);
+
+    // test input
+    const input = canvas.getByRole('textbox') as HTMLInputElement;
+    const testInputText = 'test input text';
+    userEvent.type(input, testInputText);
+    expect(input.value).toEqual(testInputText);
+
+    // test input wrapper label
+
+    expect(canvas.getByText(args.label as string));
```

Comment:
Should this (and the others like this) have a matcher function? `toBeInTheDocument()` maybe or `toBeVisible()`? Even if that's the default it just feels like it's more readable with a matcher

Reply:
I've updated them all to use `toBeVisible()` 👍